scripts/macos-setup-opam.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -x #echo on
+set -eu
+
+# Keep compile dirs to avoid recompiles
+export OPAMKEEPBUILDDIR='true'
+export OPAMREUSEBUILDDIR='true'
+export OPAMYES=1
+
+# Set term to xterm if not set
+export TERM=${TERM:-xterm}
+
+# ocaml downloading
+opam init
+eval $(opam config env)
+
+opam update
+
+SWITCH_LIST=$(opam switch list -s)
+SWITCH='4.07.1+statistical-memprof'
+
+# Check to see if we have switch
+SWITCH_FOUND=false
+for val in $SWITCH_LIST; do
+  if [ $val == $SWITCH ]; then
+    SWITCH_FOUND=true
+  fi
+done
+
+if [ "$SWITCH_FOUND" = true ]; then
+  opam switch set $SWITCH
+else
+  opam switch create $SWITCH || true
+  opam switch $SWITCH
+fi
+
+# Test -- see if curent export is different from saved export
+opam switch export opam.export.test
+diff -U1 opam.export.test src/opam.export || true
```

Comment:
how will we remember to update this?

Reply:
this is a debug/wip

I'm trying to understand if import is needed or not by looking at this diff.

I'm still not sure if import ALWAYS reinstalls or just does a delta apply.